prism/src/main/java/org/apache/falcon/resource/AbstractEntityManager.java

```diff
@@ -359,6 +364,80 @@ protected APIResult update(Entity newEntity, String type, String entityName, Boo
         }
     }
 
+    /**
+     * Updates scheduled dependent entities of a cluster.
+     *
+     * @param clusterName   Name of cluster
+     * @param colo colo
+     * @param skipDryRun Skip dry run during update if set to true
+     * @return APIResult
+     */
+    public APIResult updateClusterDependents(String clusterName, String colo, Boolean skipDryRun) {
+        checkColo(colo);
+        try {
+            verifySafemodeOperation(null, EntityUtil.ENTITY_OPERATION.UPDATE_CLUSTER_DEPENDENTS);
```

Comment:
The operation is being performed on either Feed or Process entities. But the actual entity type should not matter for the operation, Hence sending null.